src/game/Tactical/Soldier_Control.cc

```diff
@@ -7663,7 +7663,7 @@ BOOLEAN InternalIsValidStance(const SOLDIERTYPE* pSoldier, INT8 bDirection, INT8
 
 	if ( pSoldier->bCollapsed )
 	{
-		if ( bNewStance == ANIM_STAND || bNewStance == ANIM_CROUCH )
+		if ( bNewStance == ANIM_STAND || bNewStance == ANIM_CROUCH && pSoldier->bBreath < OKBREATH)
```

Comment:
I'm pretty sure you need to put parentheses around the original conditions, since && has precedence over ||. At the same time you can kill that starting space.